docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-sphinx-rtd-theme
-sphinxcontrib-napoleon
-sphinx
-plantweb
\ No newline at end of file
+sphinx-rtd-theme~=0.5.0
+sphinxcontrib-napoleon~=0.7
+sphinx~=3.2.0
+plantweb~=1.2.1
```

Comment:
@JSchripsema-IDM @devclinton is it critical that we semi-pin versions? it can cause issues on non-VM platforms so would prefer to leave unpinned if possible, and i can see this causing issues since synthpops is designed to work with other libraries that might have incompatible requirements

Reply:
Clinton can chime in too, but I was following this example as a best practice since dependbot has flagged packages that need to be updated in idmtools and other projects, but I think if it's likely to cause problems for SynthPops or Covasim we could remove.

Reply:
@cliffckerr @JSchripsema-IDM When possible it is good to pin like this(to allow updating to latest compatible version). There are cases though this can cause issues or bloat builds so you can adjust per project/dependency when necessary